duckduckgo/Android
# Add DRM to remote config

Task/Issue URL: https://app.asana.com/0/1125189844152671/1201306932455980

### Description
This PR moves the DRM manager and list into the remote privacy config.

### Steps to test this PR

1. Go to spotify.com
1. Log in with your account
1. The player should successfully show up and you should be able to play songs, etc.
